site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-siteInfo-list.vm

```diff
@@ -1,5 +1,93 @@
 ##<!-- $Header: /cvs/sakai2/legacy/tools/src/webapp/vm/sitesetup/chef_site-siteInfo-list.vm,v 1.6 2005/05/28 03:04:36 ggolden.umich.edu Exp $ -->
 
+## This macro has been defined in order to reuse the paging panel once at the top of the participant group table and once at the bottom of the participant group table. JIRA Ref -WL3736
+#macro(pagingPanel $pagesizeFormName)
+<div class="navPanel">
+	<div class="viewNav">
+		<h4>
+			$validator.escapeHtml($!siteTitle) $tlang.getString("sitegen.siteinfolist.part") ($tlang.getFormattedMessage("sinfo.list.participantListSize", $!participantListSize))
+		</h4>
+	</div>
+	<div class="listNav">
+		<div class="instruction">$tlang.getString("sitegen.siteinfolist.view") $topMsgPos - $btmMsgPos $tlang.getString("sitegen.siteinfolist.of") $allMsgNumber $tlang.getString("sitegen.siteinfolist.items")</div>
+			#if ($pagesize != 0)
+				#if ($goFPButton == "true")
+					<form name="firstpageForm" class="inlineForm" method="post" action="#toolForm("$action")">
+						<fieldset><legend>$tlang.getString("gen.first")</legend>
+							<input type="submit" name="eventSubmit_doList_first" value="|&lt;" title="$tlang.getString("gen.first")" />
+						</fieldset>
+						<input type="hidden" name="sakai_csrf_token" value="$sakai_csrf_token" />
+					</form>
+				#else
+					<form name="firstpageForm" class="inlineForm" method="post" action="#toolForm("$action")">
+						<fieldset><legend>$tlang.getString("gen.first")</legend>
+							<input type="submit" name="eventSubmit_doList_first" value="|&lt;" disabled="disabled" />
+						</fieldset>
+						<input type="hidden" name="sakai_csrf_token" value="$sakai_csrf_token" />
+					</form>
+				#end
+				#if ($goPPButton == "true")
+					<form name="prevpageForm" class="inlineForm" method="post" action="#toolForm("$action")">
+						<fieldset><legend>$tlang.getString('gen.previous') $pagesize</legend>
+							<input type="submit" name="eventSubmit_doList_prev" value="&lt;" title="$tlang.getString('gen.previous') $pagesize" />
+						</fieldset>
+						<input type="hidden" name="sakai_csrf_token" value="$sakai_csrf_token" />
+					</form>
+				#else
+					<form name="prevpageForm" class="inlineForm" method="post" action="#toolForm("$action")">
+						<fieldset><legend>$tlang.getString('gen.previous') $pagesize</legend>
+							<input type="submit" name="eventSubmit_doList_prev" value="&lt;" disabled="disabled" />
+						</fieldset>
+						<input type="hidden" name="sakai_csrf_token" value="$sakai_csrf_token" />
+					</form>
+				#end
+			#end
+			<form id="$pagesizeFormName" name="$pagesizeFormName" class="inlineForm" method="post" action="#toolForm("$action")">
+				<input type="hidden" name="eventSubmit_doChange_pagesize" value="changepagesize" />
+				<label for="selectPageSize" class="skip">$tlang.getString("sitegen.siteinfolist.view.pagesize")</label>
```

Comment:
The <label for=""> should match the <select id=""> so that the browser knows which element the label is for. The alternative is to enclose the select element inside the label, but then you don't have as much control over layout.